docroot/themes/humsci/humsci_basic/templates/menus/menu--main.html.twig

```diff
@@ -14,17 +14,20 @@
  */
 #}
 {% set toggle_text = "Menu"|t %}
-{% set attributes = attributes.addClass(['su-main-nav','su-main-nav--light', 'su-multi-menu', 'su-multi-menu--buttons']) %}
+{% set attributes = attributes.addClass(['hb-main-nav', 'su-main-nav']) %}
 {% set attributes = attributes.setAttribute('aria-label', 'main menu') %}
 {# Macros #}
 {%- import "@humsci_basic/menus/macros/nav-menu.twig" as menus -%}
 <nav{{ attributes }}>
   {% block multimenubutton %}
-  <button class="su-main-nav__toggle su-main-nav__toggle--right {{ toggle_modifier_class }}" aria-expanded="true">{{ toggle_text|default('Menu') }}</button>
+  <button class="hb-main-nav__toggle hb-main-nav__toggle--right {{ toggle_modifier_class }}" aria-expanded="true">
+    {{ toggle_text|default('Menu') }}
```

Comment:
I think this probably should no longer be named `toggle_text`. We also probably dont need a variable